### ISSUE TITLE
two question

### ISSUE THREAD

**Issue by fuyi02** [dvlab-research/ECCV22-P3AFormer-Tracking-Objects-as-Pixel-wise-Distributions#5]
1. load model, get error: "PytorchStreamReader failed reading zip archive: failed finding central directory"   load scripts is sh configs/standard/v100_test_mot17.sh
2. I can't find code about pixel-wise association, only bytetracker part,  no center points association which present in your paper., Is this code complete? 
![image](https://user-images.githubusercontent.com/39895596/182817489-ffd79d00-7d6b-47a7-b29e-d582e4b97648.png)
![image](https://user-images.githubusercontent.com/39895596/182817709-b5d4b873-369d-4abe-bc06-f453baf75aec.png)

**Comment by smart-face** [comment]
Yes, the same issue is here! The paper introduces the inspired idea from the flow-guided feature aggregation for video object detection which I could not be able to find the usage inside the code.

@dense_tracker.py file the get_flow has no usage.
from util.p3aformer.p3aformer_misc import (
    get_flow,
    NestedTensor,
    gaussian_radius,
    affine_transform,
    draw_umich_gaussian,
    soft_nms_pytorch,
)
